modules/module_manager/README.md

```diff
@@ -0,0 +1,41 @@
+# Module Manager
+
+## Purpose
+
+The Module Manager is intended to be a place for administrators
+to view and configured modules which are installed on the LORIS
+instance.
+
+## Intended Users
+
+The module is used by administrators to configure modules in
+LORIS.
+
+## Scope
+
+The module only displays and configures modules that are already
+installed (populated in the module table.)
+
+NOT in scope:
+
+The module manager does not install new modules
+
+## Permissions
+
+The `module_manager_view` permission allows a user to
+view the list of modules, and the `module_manager_edit`
+allows the user to modify the state of installed modules.
+
+## Configurations
+
+The module manager gets a list of modules from the `modules`
+database table. This table must first be populated (usually by
+using the `tools/manage_modules.php` script)
```

Comment:
```suggestion
using the `tools/manage_modules.php` script).
```

Reply:
That's not correct English grammar. The punctuation goes inside the bracket.

Reply:
I think my suggestion is correct but we don't need to quibble about it. I'll dismiss my review either way. 

https://www.grammarbook.com/punctuation/parens.asp

Reply:
Anyone can register a domain name with "grammar" in it, so I went to check the Oxford style guide and it agrees with you, so I merged the change.